rollingpin/main.py

```diff
@@ -188,8 +188,13 @@ def _main(reactor, *raw_args):
         for host in hosts:
             print host.name
     else:
-        deployer = Deployer(config, event_bus, args.parallel,
-                            args.sleeptime, args.timeout)
+        deployer = Deployer(config,
+                            event_bus,
+                            args.parallel,
+                            args.sleeptime,
+                            args.timeout,
+                            args.dangerously_fast,
+                            )
```

Comment:
Yeah, absolutely, I think dan was referring to the "or it may be lined up under the first character of the line that starts the multi-line construct, as in:" portion at the end of that section though:

```python
deployer = Deployer(a,
    b,
    c,
    d,
)  # <-- i am all the way left again
```